data/crosswalk.json

```diff
@@ -6,7 +6,7 @@
   },
   {
     "local_id": "wat-88",
-    "COMID": "9731236"
+    "COMID": "cat-88"
```

Comment:
I don't think we should modify this crosswalk, it has other implications.  A COMID should be a COMID, not a catchment id.